docs/farming-&-staking/farming/additional-guides/gpu-plotter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Experimental GPU Plotter
+sidebar_position: 4
+description: Tips on GPU Plotter
+keywords:
+    - Plotter
+    - Farming
+    - GPU
+---
+
+The first test builds of the **GPU Plotter** are now available!!!
+
+[Download GPU Plotter](https://github.com/autonomys/subspace/actions/runs/10850781628)
+
+## Overview
+The GPU Plotter will eventually support most NVIDIA and AMD video cards. For now, only NVIDIA cards are supported.
+
+The download package includes both the node and farmer Advanced CLI executables, but you only need the farmer for plotting. It works for both standalone farming/plotting and farming clusters. Support for Space Acres will be added in the future.
+
+You will need to install either the NVIDIA 550 or 560 drivers if you are using Linux.
```

Comment:
I'd suggest to have a Requirements section after the overview, so that is stands out more

Reply:
And what if I'm not using Linux? Is is the same drivers, or something different?